### PR TITLE
Fixing bugs on Rename column and Show Dialog

### PR DESCRIPTION
**Pull request by Lunalo**

### CODE REVIEW

File: instat/dlgShowModel.vb [hunk 1]
```diff
@@ -54,18 +52,14 @@ Public Class dlgShowModel
         ucrInputProbabilities.Reset()
         ucrInputNewColNameforTablePlus.Reset()
         rdoQuantiles.Checked = True
+        setname()
```
Comment (dannyparsons) [inline instat/dlgShowModel.vb]: Can you check the methods in here have the correct naming convention, capital letters etc.

**Comment by dannyparsons** [comment]
Great, can someone test?

**Comment by Lunalo** [comment]
@lilyclements 

This now ready. I will make changes for resetting ucrDistributionWithParameters in the new pull request

**Comment by lilyclements** [comment]
Great. This looks ready for Danny to review

**Pull request merged**